app/screens/settings-screen/settings-screen.tsx

```diff
@@ -32,7 +32,11 @@ const styles = EStyleSheet.create({
   },
 })
 
-export const SettingsScreen: ScreenType = ({ navigation }) => {
+type Props = {
+  navigation: any,
```

Comment:
we're just pushing the `any` one depth further here :) but that'll do it for now. thanks.